package.json

```diff
@@ -40,6 +40,7 @@
     ]
   },
   "devDependencies": {
+    "@iconify/react": "^3.1.2",
```

Comment:
Is this necessary to use this library? You can download icons from figmą project and use it as normal React component.